webapp/packages/plugin-user-profile/src/UserProfileOptionsPanelService.ts

```diff
@@ -82,7 +82,7 @@ export class UserProfileOptionsPanelService {
       return;
     }
 
-    if (this.userInfoResource.data === null) {
+    if (!this.userInfoResource.isData()) {
```

Comment:
here is better to use `!this.userInfoResource.hasAccess` so we will close this panel if user now not anonymous and not real user